### PR TITLE
feat(frontend): code changes to add env vars and make AMS URL available

### PR DESCRIPTION
**Pull request by mishraomp**
1. Added AMS(Posse) URL as env var
2. Some modifications to code have configs as env vars from hardcoded values.

---

Thanks for the PR!

Deployments, as required, will be available below:
- [Frontend](https://nr-epd-organics-info-179-frontend.apps.silver.devops.gov.bc.ca)
- [Backend](https://nr-epd-organics-info-179-frontend.apps.silver.devops.gov.bc.ca/api/)

Please create PRs in draft mode.  Mark as ready to enable:
- [Analysis Workflow](https://github.com/bcgov/nr-epd-organics-info/actions/workflows/analysis.yml)

After merge, new images are deployed in:
- [Merge Workflow](https://github.com/bcgov/nr-epd-organics-info/actions/workflows/merge.yml)

### CODE REVIEW

**Comment by mishraomp** [comment]
this one partly solves auth documents as described in https://github.com/bcgov/nr-epd-organics-info/issues/112

**Review by DerekRoberts — APPROVED** [review]

File: frontend/Caddyfile [hunk 1]
```diff
@@ -14,11 +14,17 @@
...
+          Content-Type text/javascript
+        }
+        respond `window.config={"VITE_AMS_URL":"{$AMS_URL}"};`
+    }
 	root * /srv
 	encode zstd gzip
 	file_server
 	@spa_router {
-		not path /api/*
+		not path /api/* /env.js
```
Comment (ccallendar) [inline frontend/Caddyfile:27]: Is this meant to be `/env.ts`? Or does that file get compiled to `env.js`?

**Review by ccallendar — DISMISSED** [review]
Looks good to me! I did ask one comment about the Caddyfile, but I assume what you have is correct :)

File: frontend/Caddyfile (hunk 1 above)
Comment (mishraomp) [reply frontend/Caddyfile:27]: the /env.js is a new route which will browser will call out, I will add the missing code :).

**Pull request merged**